src/Sdk/Cerbos/Sdk/Cloud/V1/HubClientBuilder.cs

```diff
@@ -32,77 +29,61 @@ public static HubClientBuilder ForTarget(string target)
             return new HubClientBuilder(target);
         }
 
-        public HubClientBuilder WithCaCertificate(string path) {
-            CaCertificate = path;
-            return this;
-        }
-
-        public HubClientBuilder WithCredentials(Credentials credentials)
-        {
-            Credentials = credentials;
-            return this;
-        }
-
         public HubClientBuilder WithCredentials(string clientId, string clientSecret)
         {
             Credentials = new Credentials(clientId, clientSecret);
             return this;
         }
 
-        public HubClientBuilder WithPlaintext()
-        {
-            Plaintext = true;
-            return this;
-        }
-
         public IHubClient Build()
         {
             if (string.IsNullOrEmpty(Target))
             {
                 throw new Exception("Target must be specified");
             }
 
-            if (CaCertificate != null && Plaintext)
-            {
-                throw new Exception("CaCertificate and plaintext must not be specified at the same time");
-            }
-
             if (Credentials == null)
             {
                 throw new Exception("Credentials must be specified");
             }
 
-            var channelOptions = new GrpcChannelOptions{};
-            if (CaCertificate != null)
-            {
-                var handler = new HttpClientHandler();
-                var cert = new X509Certificate(CaCertificate);
-                handler.ClientCertificates.Add(cert);
-                channelOptions.HttpHandler = handler;
-            }
-            else if (!Plaintext)
+            var channelOptions = new GrpcChannelOptions
             {
-                channelOptions.Credentials = ChannelCredentials.SecureSsl;
-            }
-
+                ServiceConfig = new ServiceConfig
+                {
+                    MethodConfigs = {
+                        new MethodConfig {
+                            Names = { MethodName.Default },
+                            RetryPolicy = new RetryPolicy
+                            {
+                                RetryableStatusCodes = {
+                                    StatusCode.ResourceExhausted,
```

Comment:
Let's remove `ResourceExhausted` because retrying would probably exacerbate the problem. This is the default config I used in other SDKs: https://github.com/cerbos/cerbos-sdk-python/blob/35257a6f7a7ac251c2e91a368184d233a5bd95cb/src/cerbos/sdk/_async/_hub/_client.py#L14-L44